library/spdm_common_lib/libspdm_com_crypto_service.c

```diff
@@ -20,27 +20,19 @@ bool libspdm_get_peer_cert_chain_buffer(void *context,
                                         const void **cert_chain_buffer,
                                         size_t *cert_chain_buffer_size)
 {
-    libspdm_context_t *spdm_context;
 #if LIBSPDM_RECORD_TRANSCRIPT_DATA_SUPPORT
+    libspdm_context_t *spdm_context;
     uint8_t slot_id;
-#endif
 
     spdm_context = context;
-#if LIBSPDM_RECORD_TRANSCRIPT_DATA_SUPPORT
     slot_id = spdm_context->connection_info.peer_used_cert_chain_slot_id;
-    LIBSPDM_ASSERT(slot_id < SPDM_MAX_SLOT_COUNT);
```

Comment:
why you remove this ASSERT?

Reply:
It's a miss. Thanks for pointing it out.
I'll fix it by next patch set.